cypress/support/constants/constants.js

```diff
@@ -321,7 +321,6 @@ module.exports = {
   PLATFORM_NOT_TRIGGER_EVENT: 'Platform MUST not trigger event ',
   PLATFORM_TRIGGER_EVENT: 'Platform MUST trigger event ',
   PLATFORM_RELEASE: 'Release',
-  PLATFORM: 'Platform',
   PARTNER: 'Partner',
```

Comment:
I think this got fixed in https://github.com/rdkcentral/firebolt-certification-suite/pull/179 pr

Reply:
yes, removed this change